sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -454,15 +454,16 @@ def create_snapshot( # type: ignore
 
     @distributed_trace
     def delete_share(
-            self, delete_snapshots=False, # type: Optional[bool]
-            **kwargs
-        ):
-        # type: (...) -> None
+        self, delete_snapshots: Optional[Literal['include', 'include-leased']] = False,
```

Comment:
So should I throw an ignore on this? It needs to keep it's default value of `bool` but now the typehint is non-bool, so I think everyone's static type-checker would be going to town 😄